ui-admin/src/forms/designer/NewQuestionForm.test.tsx

```diff
@@ -13,7 +14,7 @@ describe('NewQuestionForm', () => {
     const questionTypeSelect = screen.getByLabelText('Question type')
     expect(questionStableIdInput).toBeInTheDocument()
     expect(questionTypeSelect).toBeInTheDocument()
-    expect((questionTypeSelect as HTMLSelectElement).value).toBe('text')
+    expect((questionTypeSelect as HTMLSelectElement).value).toBe('placeholder')
```

Comment:
```suggestion
    expect(questionTypeSelect).toHaveValue('placeholder')
```

`@testing-library/jest-dom` (imported in setupTests) has a lot of matchers that make assertions like this a bit clearer.

https://github.com/testing-library/jest-dom#table-of-contents
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-assertion